### ISSUE TITLE
cleanup_temp_tables

### ISSUE THREAD

**Issue by manfredknapp** [utPLSQL/utPLSQL#432]
The package ut_runner uses authid current_user. If you try to execute ut3.run() you get the error "line 1: ORA-00942: Tabelle oder View nicht vorhanden
ORA-06512: in "UT3.UT_RUNNER", Zeile 49
ORA-06512: in "UT3.UT_RUNNER", Zeile 84
ORA-00942: Tabelle oder View nicht vorhanden
ORA-06512: in "UT3.UT_RUNNER", Zeile 112
ORA-06512: in "UT3.UT", Zeile 292
ORA-06512: in "UT3.UT", Zeile 267
ORA-06512: in Zeile 2

ut3 being the public synonym for ut3.ut.

The problem ist that you don't have the priv. to the table ut_cursor_data.

I moved the procedure "cleanup_temp_tables" to ut_utils. works like a charm.

**Comment by jgebal** [comment]
Hi @manfredknapp 
This was a miss in release 3.0.2 I think.
We actually spotted this already and in develop, the code is sitting in `ut_utils` also we now use truncate rather then delete on the global temp table, so the resource is free after use. 
I've noticed some issues with uninstall script failing to drop the table even though transactions ere committed if the session that was running utPLSQL was still open.
Next release will contain the fix.
Thanks for reporting it.
